sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -1079,6 +1079,42 @@ class StreamSuite extends StreamTest {
       assert(query.exception.isEmpty)
     }
   }
+
+  Seq(true, false).foreach { useV2Sink =>
```

Comment:
This structure makes it impossible to execute test independently (at least I don't know). The whole suite has to be executed which makes even small modifications time consuming.

Reply:
RIght. That was to minimize the code, but I guess only some lines will be added and it will become easier to test so better to apply your suggestion. Thanks!